Assets/RGLUnityPlugin/Scripts/LidarSensor.cs

```diff
@@ -220,11 +233,27 @@ public void Capture()
         {
             sceneManager.DoUpdate();
 
+            //Calculate delta transform of lidar.
+            Matrix4x4 currentTransform = gameObject.transform.localToWorldMatrix * configuration.GetLidarOriginTransfrom();
```

Comment:
You don't need to apply `configuration.GetLidarOriginTransfrom()`. It doesn't change the speed of the sensor.